packages/@aws-cdk/cloudformation-include/test/valid-templates.test.ts

```diff
@@ -95,33 +94,53 @@ describe('CDK Include', () => {
   test('can ingest a template with a Bucket Ref-erencing a KMS Key, and output it unchanged', () => {
     includeTestTemplate(stack, 'bucket-with-encryption-key.json');
 
-    expect(stack).toMatchTemplate(
+    Template.fromStack(stack).templateMatches(
       loadTestFileToJsObject('bucket-with-encryption-key.json'),
     );
   });
 
   test('accepts strings for properties with type number', () => {
     includeTestTemplate(stack, 'string-for-number.json');
 
-    expect(stack).toMatchTemplate(
-      loadTestFileToJsObject('string-for-number.json'),
-    );
+    Template.fromStack(stack).hasResourceProperties('AWS::S3::Bucket', {
+      "CorsConfiguration": {
+        "CorsRules": [
+          {
+            "AllowedMethods": [
+              "GET",
+            ],
+            "AllowedOrigins": [
+              "*",
+            ],
```

Comment:
Let's drop these properties from the test (only `"MaxAge"` do we actually care about in this test).